### DISCUSSION TITLE
Changing line-height of a CTkLabel (spacing1,spacing2,spacing3 on og tkinter)

### DISCUSSION THREAD

**Discussion by RVFET** [TomSchimansky/CustomTkinter#970] [Q&A] [answered]
I'd like to change the spacing between the lines of a long paragraph of text. The text is inside of a CTkLabel and afaik there's no built-in method to change line-height property (og tkinter has that, it's called spacing1,spacing2 and spacing3). Is there a way to achieve that?

![image](https://user-images.githubusercontent.com/58657896/210131366-56601896-2b17-46e9-96fc-6823ac1118f1.png)

**Comment by RVFET** [comment, marked as answer]
I found the solution, I won't delete it so future devs can google and find the answer.
I changed **CTkLabel** to **CTkTextBox** and all those properties mentioned above (spacing1, spacing2, spacing3) are working now. You may want to add ```self.textbox.configure(state="disabled")``` If you don't want it to be editable.

**Comment by Akascape** [comment]
>og tkinter has that, it's called spacing1,spacing2 and spacing3

@RVFET og **tkinter label don't have spacing options**. So CTklabel will also have no spacing option as it is made up of tkinter label only.

But the tkinter **text** widget has spacing parameters and CTkTextbox is made up of that text widget which will obviously have that spacing parameters.

In short, you have to use a tkinter text widget with customtkinter if you want to use spacings.
